### PR TITLE
Sho/await-delay

### PR DESCRIPTION
**Pull request by sfjt**
- Introducing a delay function (with await)
- prettier formatting

### CODE REVIEW

**Comment by sfjt** [comment]
@FujiHaruka 
Could you review the new JS snippet?
(including the comments)

File: content/en/snippets/others/delay.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+function delay(seconds) {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, seconds * 1000);
+  });
+}
+
+return (async () => {
+  // 処理の解決を待つために
+  // Promise オブジェクトを返す必要があります。
+  // このスコープの中にコードを書き込んでください。
```
Comment (meih) [inline content/en/snippets/others/delay.md:19]: It looks like this is a Japanese comment for the English page 🙂

File: content/ja/snippets/others/delay.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    setTimeout(resolve, seconds * 1000);
+  });
+}
+
+return (async () => {
+  // You want to return promise object
+  // so that the test executon engine can wait it to be resolved.
+  // Please write the code inside of this scope.
+
+  // Example:
```
Comment (meih) [inline content/ja/snippets/others/delay.md:21]: And this is an English comment for the Japanese page

**Review by meih — CHANGES_REQUESTED** [review]
Thank you! Just left some comments for the comments

**Comment by sfjt** [comment]
@meih Thank you for the catch. I'll fix that!

File: content/ja/snippets/others/delay.md (hunk 2 above)
Comment (sfjt) [reply content/ja/snippets/others/delay.md:21]: fixed the loc issue

File: content/en/snippets/others/delay.md (hunk 1 above)
Comment (sfjt) [reply content/en/snippets/others/delay.md:19]: fixed the loc issue

**Review by FujiHaruka — APPROVED** [review]
Thank you! Looks great to me 👍

**Pull request merged**